src/AwesomeAssertions.Web/AwesomeAssertions.Web.csproj

```diff
@@ -0,0 +1,56 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <PackageDescription>
+      This is a AwesomeAssertions extension over the HttpResponseMessage object.
+      It provides specific assertions and outputs rich erros messages that include the HTTP Request and Response.
+      This version depends on the commercial version of the FluentAssertions library (&gt;= 8.0.0).
+    </PackageDescription>
+    <PackageReadmeFile>readme.md</PackageReadmeFile>
+  </PropertyGroup>
+
+  <PropertyGroup>
+    <DefineConstants>$(DefineConstants);FAV8;AAV</DefineConstants>
```

Comment:
Not sure if I understand this part. If FAV8 is set, then it means we also compile code specific to FAV8. This shouldn't be set, only AAV.

Reply:
Ah, ok, this is because AwesomeAssertions is based on the last commit before the license change, got it.